src/ufo-measure-task.c

```diff
@@ -28,16 +35,49 @@
  */
 
 typedef enum {
-    M_0,
-    M_STD,
     M_MIN,
     M_MAX,
-    M_LAST
+    M_SUM,
+    M_MEAN,
+    M_VAR,
+    M_STD,
+    M_SKEW,
+    M_KURTOSIS,
```

Comment:
Could you still use `M_LAST` here and in the static data structures? It's easier to understand and in the long run easier to extend.